libraries/lib-time-track/TimeTrack.cpp

```diff
@@ -222,7 +222,8 @@ void TimeTrack::Paste(double t, const Track &src)
       (void)0;// intentionally do nothing.
 }
 
-void TimeTrack::Silence(double WXUNUSED(t0), double WXUNUSED(t1))
+void TimeTrack::Silence(
+   double WXUNUSED(t0), double WXUNUSED(t1), ProgressReporter)
```

Comment:
Eliminate `WXUNUSED` when you have an excuse to disturb a function with that macro.
Just declare `double, double`

Or if you want to be really modern: https://en.cppreference.com/w/cpp/language/attributes/maybe_unused